MPU/include/pedals.h

```diff
@@ -81,6 +83,20 @@ class Pedals
          * @return If the accelerators are not faulted
          */
         FaultStatus_t readAccel();
+
+        /**
+         * @brief retrieves the value of the torque percentage
+         * 
+         * @return int representing the percentage of torque limiting
+        */
+        uint8_t getTorqueLimitPercentage();
+
+        /**
+         * @brief Sets the percentage of torque limiting
+         * 
+         * @param percentage 
+        */
+        void setTorqueLimitPercentage(float percentage);

```

Comment:
echoing again idk why the param is a float, we try and avoid floats at all costs. Can we shift it to be *100 or something to make it not a float? Might be confused on the use too

Reply:
I initally had it as an int but @mccauleyma said that it doesnt make sense to multiply by 100 just to divide by 100 when we use it

Reply:
eh whatever lol